Tests/test_excinfo.py

```diff
@@ -538,10 +465,10 @@ def __exit__(self, t, v, tb):
             with M2(self):
                 self.A(15)
                 raise ValueError(34)
-            if is_cli:  # http://ironpython.codeplex.com/workitem/27990
-                self.A(15)
-            else:
-                self.A(34)
+            self.A(15) # TODO: is this a bug?
+
 
+#run_test(__name__)
 
-run_test(__name__)
\ No newline at end of file
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
Do these changes make the test pass? If so we should remove the ignore for `test_excinfo` in `IronPythonCasesManifest.ini`. Also, did you try it with CPython?

Reply:
Yes -- I ran it with Python 3.6 and that's how I knew what needed to change inside the test vs what needed to change in IronPython. The changes also get test_excinfo passing.